### PR TITLE
Cleanup 2023

### PR DESCRIPTION
**Pull request by dron0gus**

### CODE REVIEW

File: firmware/hw_layer/adc/adc_inputs.cpp [hunk 1]
```diff
@@ -398,6 +398,7 @@ class SlowAdcController : public PeriodicController<256> {
 };
 
 void addChannel(const char *name, adc_channel_e setting, adc_channel_mode_e mode) {
+	(void)name;
```
Comment (mck1117) [inline firmware/hw_layer/adc/adc_inputs.cpp:401]: why not just remove the parameter?

Comment (dron0gus) [reply firmware/hw_layer/adc/adc_inputs.cpp:401]: it is used in many places. not sure about long term plans. may be we will have adc users list like we have for gpio...

File: firmware/ext/FatFS/ff.c [hunk 2]
```diff
@@ -3094,7 +3094,7 @@ FRESULT find_volume (	/* FR_OK(0): successful, !=0: any error occurred */
 	/* Find an FAT partition on the drive. Supports only generic partitioning rules, FDISK and SFD. */
 	bsect = 0;
 	fmt = check_fs(fs, bsect);			/* Load sector 0 and check if it is an FAT-VBR as SFD */
-	if (fmt == 2 || (fmt < 2 && LD2PT(vol) != 0)) {	/* Not an FAT-VBR or forced partition number */
+	if (fmt == 2 || ((fmt < 2) && (bool)(LD2PT(vol) != 0))) {	/* Not an FAT-VBR or forced partition number */
```
Comment (mck1117) [inline firmware/ext/FatFS/ff.c:3097]: why do we edit imported libraries?

Comment (rusefillc) [reply firmware/ext/FatFS/ff.c:3097]: Git submodule but yes

**Pull request closed without merging**